extension/app/main.tsx

```diff
@@ -19,7 +20,9 @@ const App = () => {
   return (
     <AuthProvider>
       <Notification.Area>
-        <RouterProvider router={router} />
+        <GenerationContextProvider>
```

Comment:
nit : shouldn't we add this in ConversationContainer instead ?

Reply:
done